--- origsrc/mpd-0.19.19/configure.ac	2016-08-15 03:00:15.000000000 -0500
+++ src/mpd-0.19.19/configure.ac	2016-11-16 22:12:23.008048300 -0600
@@ -1358,10 +1358,11 @@ if test x$enable_sidplay != xno && test
 fi
 
 if test x$enable_sidplay = xyes; then
-	SIDPLAY_LIBS="$SIDPLAY_LIBS -lresid-builder"
 	AC_DEFINE(ENABLE_SIDPLAY, 1, [Define for libsidplay2 support])
 	if test x$found_sidplayfp = xyes; then
 		AC_DEFINE(HAVE_SIDPLAYFP, 1, [Define if libsidplayfp is used instead of libsidplay2])
+	else
+		SIDPLAY_LIBS="$SIDPLAY_LIBS -lresid-builder"
 	fi
 fi
 
